accounts-db/src/accounts_db/geyser_plugin_utils.rs

```diff
@@ -141,7 +141,7 @@ pub mod tests {
     };
 
     impl AccountsDb {
-        pub fn set_geyser_plugin_notifer(&mut self, notifier: Option<AccountsUpdateNotifier>) {
+        pub fn set_geyser_plugin_notifier(&mut self, notifier: Option<AccountsUpdateNotifier>) {
```

Comment:
can not change public API without a version bump

Reply:
Correct. However, we can do the following:
- Add a new method without the spelling mistake (`set_geyser_plugin_notifier`)
- Update the old method `set_geyser_plugin_notifer` to simply call the new method `set_geyser_plugin_notifier`
- Add a deprecation warning to the old method, including mention that the function has been deprecated in v2.3

We can then rip out the function with incorrect spelling at a later date once we are shipping a sufficiently new version